### ISSUE TITLE
HTTP 404 Not found - Invalid utility ID: system-messages

### ISSUE THREAD

**Issue by pejeio** [verbb/comments#223]
**Description**
When clicking the **System messages** link in the Comments Settings, a 404 error is shown.

<img width="1281" alt="1" src="https://user-images.githubusercontent.com/33237012/115106518-92e61e80-9f65-11eb-9f43-5a8ca11fec1b.png">

**Steps to reproduce**
1. In the CraftCMS admin panel, go to **"Comments"** in the left sidebar
2. Click **"Notifications"** --> "Settings"
3. Click on the **System messages** link, found in this text: _Authors receive an email for every new comment. Edit the content of this email through System Messages._

**Additional info**
- Plugin version: 1.8.6
- Craft version: 3.6.12
- Multi-site: Yes

**Comment by engram-design** [comment]
Do you perhaps not have permission to Utilities > System Messages? There's nothing wrong with this link, I don't believe.

**Comment by pejeio** [comment]
@engram-design I'm logged in as administrator.

**Comment by engram-design** [comment]
Can you confirm the URL it's pointing to? It should be `http://your.site/admin/utilities/system-messages`.

Can you navigate to Utilities > System Messages?

**Comment by pejeio** [comment]
@engram-design I'm not able to navigate via **Utilities** -> **System messages**

<img width="1273" alt="Schermafbeelding 2021-04-17 om 13 02 22" src="https://user-images.githubusercontent.com/33237012/115110738-28d97380-9f7d-11eb-9a97-d42f2907fd9e.png">

**Comment by engram-design** [comment]
I suspect you might be on Craft Solo? It looks like this is only possible to edit these in the control panel with Craft Pro.

You'll need to override these manually, using [static translations](https://craftcms.com/docs/3.x/sites.html#static-message-translations) by creating a `comments.php` file over altering the [translations](https://github.com/verbb/comments/blob/craft-3/src/translations/en/comments.php)

**Comment by pejeio** [comment]
@engram-design Exactly, I'm on Craft Solo.

I was trying to enable email notifications for admins. Unfortunately, no email is sent when someone posts a new comment.
Sending a test mail from `/settings/email` does, however, trigger a new email. So it has something to do with the plugin configuration I guess.

<img width="1010" alt="2" src="https://user-images.githubusercontent.com/33237012/115231204-93251c00-a115-11eb-8548-c901644b24d1.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/comments` requirement in `composer.json` to:

```
"require": {
  "verbb/comments": "dev-craft-3 as 1.8.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by pejeio** [comment]
@engram-design Can you tell when the next release with this fix will be?

**Comment by engram-design** [comment]
Fixed in [1.8.7](https://github.com/verbb/comments/releases/tag/1.8.7)

**Comment by pejeio** [comment]
Thanks @engram-design!
